var/spack/repos/builtin/packages/util-macros/package.py

```diff
@@ -11,13 +11,19 @@ class UtilMacros(AutotoolsPackage, XorgPackage):
     other Xorg modular packages, and is needed to generate new versions
     of their configure scripts with autoconf."""
 
-    homepage = "https://cgit.freedesktop.org/xorg/util/macros/"
-    xorg_mirror_path = "util/util-macros-1.19.1.tar.bz2"
+    homepage = "https://gitlab.freedesktop.org/xorg/util/macros"
+    xorg_mirror_path = "util/util-macros-1.19.1.tar.xz"
 
-    maintainers("robert-mijakovic")
+    # note: url_for_version can only return a single url, no mirrors
+    def url_for_version(self, version):
+        if self.spec.satisfies("@:1.19"):
+            return self.urls[0].replace("xz", "bz2")
```

Comment:
This does not appear to be the right solution for the existing versions of the package.

I cannot confirm the sha256s for 1.19.1-1.19.3 with this PR.  I get `Error: Could not find any remote versions for util-macros` when I try `1.19.3`.

Reply:
Likely same issue as https://github.com/spack/spack/pull/44393#issuecomment-2135950630. I'll file a fix against this package with priority.